twilight-gateway/src/ratelimiter.rs

```diff
@@ -58,8 +55,8 @@ impl CommandRatelimiter {
 
     /// Duration until the next permit is available.
     pub fn next_available(&self) -> Duration {
```

Comment:
This should arguably return the instant at which the next permit is available. Users can get the Duration until then via `elapsed.saturating_duration_since(Instant::now())`.